### ISSUE TITLE
Error when using with docker-usd

### ISSUE THREAD

**Issue by dbathily** [kcoley/gltf2usd#78]
Hi,

When I use your tool with docker-usd (`usd-docker/usd:0.8.1-centos7`) I have this error:

```
Traceback (most recent call last):
  File "/gltf2usd-master/Source/gltf2usd.py", line 728, in <module>
    convert_to_usd(args.gltf_file, args.usd_file, args.fps, args.scale, args.arkit, args.verbose, args.use_euler_rotation)
  File "/gltf2usd-master/Source/gltf2usd.py", line 681, in convert_to_usd
    usd = GLTF2USD(gltf_file=gltf_file, usd_file=usd_file, fps=fps, scale=scale, verbose=verbose, use_euler_rotation=use_euler_rotation)
  File "/gltf2usd-master/Source/gltf2usd.py", line 71, in __init__
    self.convert()
  File "/gltf2usd-master/Source/gltf2usd.py", line 659, in convert
    self.convert_nodes_to_xform()
  File "/gltf2usd-master/Source/gltf2usd.py", line 87, in convert_nodes_to_xform
    self._convert_node_to_xform(node, parent_transform)
  File "/gltf2usd-master/Source/gltf2usd.py", line 99, in _convert_node_to_xform
    xformPrim.AddTransformOp().Set(self._compute_rest_matrix(node))
  File "/gltf2usd-master/Source/gltf2usd.py", line 569, in _compute_rest_matrix
    return UsdSkel.MakeTransform(usd_translation, usd_rotation, usd_scale)
AttributeError: 'module' object has no attribute 'MakeTransform'
```

I don't have this with the prebuilt Apple use distribution. An incompatibility with docker-usd?

Thank you for your job

**Comment by kcoley** [comment]
Hi @dbathily , do you know which version of USD the docker image is using? gltf2usd was built against v18.09 which added skeletal animation support.

**Comment by dbathily** [comment]
According to the code source [https://github.com/AnimalLogic/docker-usd/blob/70e71d9e6101fdd10fe9a43c82f4d30eb0c4f5a7/linux/centos7/usd/Dockerfile#L9](https://github.com/AnimalLogic/docker-usd/blob/70e71d9e6101fdd10fe9a43c82f4d30eb0c4f5a7/linux/centos7/usd/Dockerfile#L9) it's `18.09`

**Comment by dbathily** [comment]
@kcoley I think you are right. In fact he just merged the "18.09" branch. I'm rebuilding the container
